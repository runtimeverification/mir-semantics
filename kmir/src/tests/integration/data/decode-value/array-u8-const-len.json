{
  "bytes": [
    0,
    1,
    2,
    3
  ],
  "types": [
    [
      0,
      {
        "PrimitiveType": {
          "Uint": "U8"
        }
      }
    ]
  ],
  "typeInfo": {
    "ArrayType": {
      "elem_type": 0,
      "size": {
        "id": 100,
        "kind": {
          "Value": [
            0,
            {
              "bytes": [
                4
              ],
              "provenance": {
                "ptrs": []
              },
              "align": 1,
              "mutability": "Not"
            }
          ]
        }
      }
    }
  }
}
