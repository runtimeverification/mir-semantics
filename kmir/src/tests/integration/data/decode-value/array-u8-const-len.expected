Range ( ListItem ( Integer ( 0 , 8 , false ) )
ListItem ( Integer ( 1 , 8 , false ) )
ListItem ( Integer ( 2 , 8 , false ) )
ListItem ( Integer ( 3 , 8 , false ) ) )
